assets/translations/en-us.yaml

```
@@ -782,6 +784,12 @@ monitoring:
       retentionSize: Retention Size
       resourceLimits: Resource Limits
       scrape: Scrape Interval
+      ignoreNamespaceSelectors:
+        label: Use Namespace Selectors
+        radio:
+          enforced: "Use: Monitors can access resources in the namespace specified in the namespaceSelector field"
```

```suggestion
          enforced: "Use: ServiceMonitors & PodMonitors can be configured to monitor resources across namespaces using the `namespaceSelector` field"
```

Not sure if the \` works as expected, but would be nice if we can highlight `namespaceSelector` to make it clear that it is a field of the ServiceMonitor / PodMonitor spec